services/mysql.rst

```diff
@@ -1,17 +1,16 @@
 MySQL/MariaDB
 =============
 
-`MariaDB`_ is a MySQL-compatible relational database system. Its XtraDB storage
-engine is equivalent to MySQL with InnoDB.
+`MariaDB`_ is a MySQL-compatible relational database system.
 
 To use it in your application, add it to your ``.symfony/services.yaml``:
 
 .. code-block:: yaml
 
     # .symfony/services.yaml
     mydatabase:
-        # supported versions: 10.0, 10.1, 10.2, 10.3, 10.4, 10.5
-        # 5.5 is also available but not maintained upstream
+        # supported versions: 10.2, 10.3, 10.4, 10.5
+        # 5.5, 10.0 and 10.1 are also available but not maintained upstream
```

Comment:
This is wrong as far as SymfonyCloud is concerned. All these versions are still supported on our platform.

Reply:
I suggest clarifying what "upstream" means in this context. MariaDB does not support 5.5, 10.0 or 10.1 any longer.

Reply:
Understood!